api/v1beta1/cryostat_types.go

```diff
@@ -90,6 +90,10 @@ type CryostatSpec struct {
 	// +optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	Resources ResourceConfigList `json:"resources,omitempty"`
+	// A permission mapping from Cryostat resources to Kubernetes resources used for authorization.
+	// +optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	AuthProperties AuthPropertiesConfigMap `json:"authProperties,omitempty"`
```

Comment:
Maybe add a `displayName` attribute here, giving the full "Authorization Properties" name in the UI. See `MaxWsConnections` as an example.